### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by sevein**
This pull request updates urllib3 to 1.26. This requires updating requests and boto3 too.

Updating urllib3 to 1.26 needed a small compatibility fix in `duracloud.py` so we can keep generating the exact same URLs bypassing the new URL parser they've implemented.

underscore.js updated to 1.13.1 - the browser continues to work properly.

Connects to https://github.com/archivematica/Issues/issues/1317.

### CODE REVIEW

File: requirements/base.in [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Base requirements - for all installations
 bagit==1.7.0
-boto3==1.9.174
-botocore==1.12.253
+boto3>=1.17.88,<1.18.0
+botocore
```
Comment (replaceafill) [inline requirements/base.in]: `botocore` doesn't need to be pinned anymore?

**Review by replaceafill — APPROVED** [review]
LGTM.

File: requirements/base.in (hunk 1 above)
Comment (sevein) [reply requirements/base.in]: I wasn't sure to what version, it's boto3 that have a hard constrain. What would you do?

Comment (replaceafill) [reply requirements/base.in]: Leave `botocore` out of the `.in` file and let `pip-tools` to deal with it in the `.txt` files?

Comment (sevein) [reply requirements/base.in]: I can do that, thanks!

**Pull request merged**